src/Entity/Builder/GenreBuilder.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * This file is part of coisa/movie-list.
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ *
+ * @link      https://github.com/coisa/movie-list
+ * @copyright Copyright (c) 2019 Felipe Sayão Lobato Abreu <github@felipeabreu.com.br>
+ * @license   https://opensource.org/licenses/MIT MIT License
+ */
+
+declare(strict_types=1);
+
+namespace CoiSA\MovieList\Entity\Builder;
+
+use CoiSA\MovieList\Entity\Genre;
+
+/**
+ * Class GenreBuilder
+ *
+ * @package CoiSA\MovieList\Entity\Builder
+ */
+class GenreBuilder implements EntityBuilderInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function createFromArray(array $result): Genre
+    {
+        return new Genre(
+            $result['id'] ?? null,
```

Comment:
This was intentional to force TypeError.

For some reason scrutinizer behaves different compared to local test running `docker-compose run --rm phpunit` so I made this change to not failing the test. In a normal situation it will never occurs.

Reply:
Related to this commit https://github.com/coisa/movie-list/pull/1/commits/04ec10a85a26df2ee71f158d9066deaa341c25e4